_translator-files/po/de/Running-a-Server.po

```diff
@@ -179,27 +214,44 @@ msgstr ""
+msgid ""
+"To run a headless server on Linux, the following steps assume you are "
+"familiar with the command line and Debian/Ubuntu or similar distribution "
+"that uses systemd."
 msgstr ""
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:62
 #, fuzzy
-#| msgid "Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) or, if you use a Raspberry Pi etc. download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})"
-msgid "If you want to run a Server on a **Raspberry Pi** (or a different armhf/arm64 Debian-based computer), you will need to download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }}) depending on your CPU - not the default `amd64` ones for use on Intel/AMD machines."
-msgstr "Lade die [neueste headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) oder, wenn du einen Raspberry Pi etc. verwendest [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})"
+msgid ""
+"If you want to run a Server on a **Raspberry Pi** (or a different armhf/"
+"arm64 Debian-based computer), you will need to download the [latest armhf ."
+"deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-"
+"headless-armhf }}) or [latest arm64 .deb file]({{ site.download_root_link }}"
+"{{ site.download_file_names.deb-headless-arm64 }}) depending on your CPU - "
+"not the default `amd64` ones for use on Intel/AMD machines."
+msgstr ""
+"Lade die [neueste headless (amd64) .deb file]({{ site.download_root_link }}"
+"{{ site.download_file_names.deb-headless }}) oder, wenn du einen Raspberry "
+"Pi etc. verwendest [latest armhf .deb file]({{ site.download_root_link }}"
+"{{ site.download_file_names.deb-headless-armhf }})"
 
 #. type: Bullet: '1. '
 #: ../wiki/en/Running-a-Server.md:66
-msgid "Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) (or for Raspberry Pi etc. as above)"
+msgid ""
+"Download the [latest headless (amd64) .deb file]({{ site."
+"download_root_link }}{{ site.download_file_names.deb-headless }}) (or for "
+"Raspberry Pi etc. as above)"
 msgstr ""
 
 #. type: Bullet: '1. '
 #: ../wiki/en/Running-a-Server.md:66
 #, fuzzy
-#| msgid "Update apt to make sure you have a current list of standard packages: `sudo apt update`"
-msgid "Make sure you have a current list of standard packages: `sudo apt update`"
-msgstr "Aktualisiere apt, um sicherzustellen, dass die Liste der Standardpakete aktuell ist: sudo apt update"
+msgid ""
+"Make sure you have a current list of standard packages: `sudo apt update`"
```

Comment:
... current list of standard packages isn't correct. Dropping standard is probably best, as we don't know what a "standard" package is.